### DISCUSSION TITLE
Taylor diagram

### DISCUSSION THREAD

**Discussion by dhithu** [DHI/modelskill#77] [Ideas]
I have taken a look at the fmskill Taylor diagram, which is a great feature! As I understand the use of the diagram is restricted to 1 observation and 1 or several models, since it is not (as far as I can see) possible to "normalize" the diagram. Since we often have 1 large model with many observation stations and parameters (e.g. the DKBS2 model), it would be great to be able to show each station or each parameter as a point in the diagram. Any plans or thoughts on this?

**Comment by jsmariegaard** [comment]
Thanks @dhithu. It's true that several observations will be shown as one aggregated point (per model). Your suggestion about normalizing with obs std makes it possible to show several observations. It is now being addressed in #79

**Comment by jsmariegaard** [comment]
#79 have now been merged - see e.g. documentation of https://dhi.github.io/fmskill/api.html#fmskill.comparison.ComparerCollection.taylor 

So I will consider this closed :-)
